service/app/util/TelemetryRequestValidator.java

```diff
@@ -85,4 +93,32 @@ private static ProjectCommonException createProjectException(String message) {
     return new ProjectCommonException(
         ResponseCode.invalidRequestData.getErrorCode(), message, ERROR_CODE);
   }
+
+  /**
+   * This method will do the check for telemetry request structure.Request can come in two format
```

Comment:
This method checks telemetry request structure. Request can come in two formats: 
Format 1: {"events":[...]}, 
Format 2: {"request":{"events":[...]}}

requestMap Telemetry request
True, if events is present in one of the mentioned formats. Otherwise, return false.